tests/test_commands.py

```diff
@@ -1,15 +1,60 @@
-'''Test Figures Django management commands
-
-'''
+"""Test Figures Django management commands
+"""
 
+import mock
+import pytest
 from django.core.management import call_command
 from django.test import TestCase
 from django.utils.six import StringIO
 
+
 class PopulateFiguresMetricsTest(TestCase):
+
     def test_command_output(self):
         out = StringIO()
         call_command('populate_figures_metrics', '--no-delay', stdout=out)
 
         self.assertEqual('', out.getvalue())
-        #self.assertIn('Expected output', out.getvalue())
\ No newline at end of file
+
+
+def test_mau_no_delay(monkeypatch):
```

Comment:
```suggestion
```

Thanks for documenting this, please see below. I think I've fixed the original function, so this one isn't needed.